packages/components/src/components/DetailsPanelList.vue

```diff
@@ -96,44 +96,45 @@ export default {
 
 <style scoped lang="scss">
 .v-details-panel-list {
-  list-style-type: none;
-  padding: 0;
-  margin: 0 -1rem 1.5rem;
+  margin-bottom: 1rem;
+  border-radius: $border-radius;
+  padding: 0.5rem;
+  color: $base03;
+  background-color: rgba(0, 0, 0, 0.1);
```

Comment:
This style is duplicated in many different components. Can it be moved into it's own component? The Figma contains guidelines for what should be componentized.

Reply:
I moved that unit test to `VsCodeExtension.spec.js` https://github.com/applandinc/appmap-js/pull/550/commits/e5bd8d3fdb624e9b23e5a67afd6d4970b55952f5